packages/core-graphql/package.json

```diff
@@ -9,7 +9,7 @@
   "main": "lib/index.js",
   "scripts": {
     "build:docs": "../../node_modules/.bin/jsdoc -c jsdoc.json",
-    "test": "cross-env ARK_ENV=test jest --forceExit --runInBand --detectOpenHandles",
+    "test": "cross-env ARK_ENV=test jest --runInBand --forceExit",
```

Comment:
This is still wrong, `forceExit` is replaced by `detectOpenHandles`.

Reply:
what do you suggest?

Reply:
As I said, use `--detectOpenHandles` instead of `--forceExit`.

Reply:
works locally now with proper plugin management (doesnt hang with `detectopenhandles`)